src/ol/render/canvas/canvasreplay.js

```diff
@@ -396,6 +397,35 @@ ol.render.canvas.Replay.prototype.replay_ = function(
           if (scale != 1 || rotation !== 0) {
             context.setTransform(1, 0, 0, 1, 0, 0);
           }
+          */
+
+          var fontSize = parseFloat(context.font);
+          var tabText = text.split('\n');
+          var cptText = tabText.length;
+
+          var newY = y;
+
+          newY = newY - (((cptText - 1) / 2) * fontSize);
```

Comment:
I think the above would be more legible with something like this:
```js
var fontSize = parseFloat(context.font);
var lines = text.split('\n');
var numLines = lines.length;
var lineY = y - (((numLines - 1) / 2) * fontSize);
```

And then I'd probably use `lineIndex` (instead of `iTxt`) and `line` (instead of `newText`) below.

The compiler makes names small for us.  So it's nice to make them legible in the source.